src/main/java/DukeException.java

```diff
@@ -0,0 +1,6 @@
+public class DukeException extends Exception{
```

Comment:
This class is well written with good checkstyle. Perhaps add a short javadoc so its easier for you to revisit it in the future.